src/components/LocationSearch/useNominatimAutocomplete.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import debounce from 'lodash/debounce';
+
+const useNominatimAutocomplete = (input) => {
+  const [places, setPlaces] = React.useState([]);
+
+  const fetchPlaces = React.useCallback(
+    debounce(async (input) => {
+      const fetchUrl = `https://nominatim.openstreetmap.org/search?q=${input}&limit=5&format=json`;
```

Comment:
This seems to work nicely.

I wonder if we can decrease the noise in the results by adding `countrycodes=gb` as seen here https://nominatim.org/release-docs/develop/api/Search/#result-limitation